src/routing/AppRoutes.tsx

```diff
@@ -100,14 +100,22 @@ const AppRoutes = (props: AppRoutesProps) => {
         </Route>
         <Route element={<RoleBAC rolesAllowed={[USER_ROLES.dataSubmitter, USER_ROLES.chairperson, USER_ROLES.admin]} />}>
           <Route path="/dataset_submissions" element={<DatasetSubmissions />} />
-          <Route path="/data_submission_form" element={<DataSubmissionForm />} />
-          <Route element={<EnvRoute env={envGroups.NON_STAGING} />}>
-            <Route path="/data_submission_form2" element={<DataSubmissionFormV2 />}>
-              <Route path=":studyId" element={<DataSubmissionFormV2 />} />
-            </Route>
-          </Route>
+          {checkEnv(envGroups.NON_STAGING) && (
```

Comment:
Nit: I naively assumed "production" would be part of `NON_STAGING`.  But I see it's `['local', 'dev']`, which is also equivalent with `DEV`.  

The latter seems more intuitive; perhaps it's worth using here.  I defer to you!


```suggestion
          {checkEnv(envGroups.DEV) && (
```

Reply:
@eweitz - Great point!  We already had a usage of NON_STAGING in this file, so I kept it consistent.  It might be worth a larger refactor around the name 'NON_STAGING' throughout the codebase.  I'll make a ticket and we can discuss at refinement.